chia/protocols/wallet_protocol.py

```diff
@@ -140,6 +140,28 @@ class RequestHeaderBlocks(Streamable):
     end_height: uint32
 
 
+@streamable
+@dataclass(frozen=True)
+class RequestHeaderBlockBlobs(Streamable):
+    start_height: uint32
+    end_height: uint32
+
+
+@streamable
+@dataclass(frozen=True)
+class RespondHeaderBlockBlobs(Streamable):
+    start_height: uint32
+    end_height: uint32
+    header_block_blobs: List[bytes]
```

Comment:
Everything in streamable is already streamed as bytes so we don't need to use bytes I  the protocol message, we can use header blocks. As long as we return the bytes verbatim from DB without calling HeaderBlock constructors

Reply:
The response object can be the same